### ISSUE TITLE
No Ethernet connection in Fedora KDE

### ISSUE THREAD

**Issue by wessel-novacustom** [Dasharo/dasharo-issues#910]
### Component

Dasharo firmware

### Device

NovaCustom V56 14th Gen

### Dasharo version

v0.9.0-rc5

### Dasharo Tools Suite version

N/A

### Brief summary

The Ethernet cable is not being recognised, a network connection can only be established via WiFi.

### How reproducible

~~100% reproducible.~~ _Edit: **not reproducible** after checking on another chassis._

### How to reproduce

1. Install Fedora 40 (KDE Plasma) on the V560TU (Ultra 7).
2. Install all available updates by using WiFi.
3. Restart the laptop and check the internet connection via cable, without using WiFi.

### Expected behavior

The network connection is established automatically without any issue.

### Actual behavior

There is no network connection (only WiFi), and it is not listed under the network settings.

### Screenshots

![v560tu_v0 9 0-rc5-no-Ethernet](https://github.com/Dasharo/dasharo-issues/assets/100232604/21ee1a3d-a0c0-4f6a-9643-1a9bee602a2f)

### Additional context

Using Fedora (KDE Plasma), kernel 6.9.5-200.fc40.x86_64

### Solutions you've tried

Nothing. I _suspect_ some issue related to the firmware, but I'm not sure. Does it make sense to try another distro?

**Comment by macpijan** [comment]
Can you check if the MAC address seems valid?
No need to post it here, but you can check it with e.g. `ip a` and check if it seems like a valid address for interface.

Definitely iPXE test would make sense -> go to iPXE shell and execute `dhcp net0`

**Comment by wessel-novacustom** [comment]
I have noticed that the **problem didn't happen** on another laptop (same config) in a live environment of the same OS.

Also, `ip a` showed a normal MAC address for the Ethernet adaptor (`D4:93:...`) for that laptop.

With that same other laptop, I have created the output in a live environment for `sudo dmesg`: [live-sudo-dmesg-output.txt](https://github.com/user-attachments/files/16018834/live-sudo-dmesg-output.txt)

For the same laptop and same live environment, the kernel was: `6.8.5-301.fc40.x86_64`.

After the installation on that other laptop, and after running all updates and restarting, the kernel version was `6.9.5-200.fc40.x86_64`, and the problem still didn't happen.

So I guess the first laptop for which this ticket was opened either has a hardware problem, the Ethernet cable was not correctly connected to the router, or the initial deployment hasn't been done correctly. It's worth saying that that laptop had `rc4` first and we upgraded to `rc5`, but it was with external flashing. I cannot believe we missed flashing only the `fd`, `me` and `bios` regions, because I cannot find an incorrect flashing command in my terminal history.

Since it's not possible to reproduce the issue, I close it.
